### PR TITLE
Don't copy over the SSH public key to the remote side

### PR DESCRIPTION
**Pull request by brianhlin**
We do this out-of-band with the site admin. We'll fix this properly later by adding another option to skip the remote copy with a PR to HTCondor but we want this fixed now.

Based off of: https://github.com/brianhlin/htcondor/tree/bosco_cluster.skip-pubkey-transfer

### CODE REVIEW

**Comment by brianhlin** [comment]
CC @jdost321 @mmascher

File: bosco_cluster [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+                       ;;
+            redhat7*) echo "`$findplatform_path --url --force RH7`"
+                       ;;
+            centos7*) echo "`$findplatform_path --url --force RH7`"
+                       ;;
+            debian6*) echo "`$findplatform_path --url --force DEB6`"
+                       ;;
+            debian7*) echo "`$findplatform_path --url --force DEB7`"
+                       ;;
+            Ubuntu12*) echo "`$findplatform_path --url --force UBUNTU12`"
```
Comment (matyasselmeci) [inline bosco_cluster:273]: Ubuntu 12???

**Comment by matyasselmeci** [comment]
Would it be possible to split this into two commits, one for adding bosco_cluster, and one for your changes to it? I don't want to review all of bosco_cluster.

File: 30-remote-site-setup.sh [hunk 2]
```diff
@@ -98,7 +98,7 @@ for ruser in $users; do
     setup_ssh_config
     [[ $cvmfs_wn_client -eq 0 ]] || setup_endpoints_ini
     # $REMOTE_BATCH needs to be specified in the environment
-    bosco_cluster "${override_opts[@]}" -a "${ruser}@$REMOTE_HOST" "$REMOTE_BATCH"
+    bosco_cluster "${override_opts[@]}" -k $"BOSCO_KEY" -a "${ruser}@$REMOTE_HOST" "$REMOTE_BATCH"
```
Comment (matyasselmeci) [inline 30-remote-site-setup.sh:101]:
```suggestion
    bosco_cluster "${override_opts[@]}" -k "$BOSCO_KEY" -a "${ruser}@$REMOTE_HOST" "$REMOTE_BATCH"
```

**Pull request merged**